common/system/tasks/radio_manager.cpp

```diff
@@ -69,6 +67,7 @@ void radioManager_tick() {
 }
 
 void radioManager_sendInternal(int radioId) {
+  strncpy(transmitPacket[radioId].callsign, cli_getConfigs()->callsign, 8);
```

Comment:
You set this in init, too. Do we need both?

Reply:
Hahaha fun fact yes because it doesn't work otherwise. We think maybe the cli_getConfigs struct gets changed after the radioManager_init happens? Or something. This makes it work even though I really don't love it...

Reply:
I guess that makes sense. Cursed.